sc-poc/managers/configman/operations.go

```diff
@@ -0,0 +1,11 @@
+package configman
+
+import (
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// List returns all the registered sources of a particular source type
+func List(gvr schema.GroupVersionResource) ([]*unstructured.Unstructured, error) {
```

Comment:
This should return the object in the same format k8s would

Reply:
Can we change ConfigType from `map[string][]*unstructured.Unstructured` to `map[string]*unstructured.UnstructuredList`? What are you thoughts on this?

Reply:
I suggest we keep the ConfigType the same but return back the response as `*unstructured.UnstructuredList`